models/intel/text-to-speech-en-0001/accuracy-check.yml

```diff
@@ -0,0 +1,37 @@
+evaluations:
+  - name: text-to-speech-en-0001
+    module: custom_evaluators.text_to_speech_evaluator.TextToSpeechEvaluator
+    module_config:
+      network_info:
+        forward_tacotron_duration: {}
+
+        forward_tacotron_regression: {}
+        forward_tacotron_regression_inputs:
+          data: data
+          data_mask: data_mask
+          pos_mask: pos_mask
+
+        melgan: {}
+        max_mel_len: '128'
+        max_regression_len: '512'
+        pos_mask_window: '4'
+
+        adapter:
+          type: regression
+          keep_shape: True
+
+      launchers:
+        - framework: dlsdk
+
+      datasets:
+        - name: forward_tacotron_melgan_io
+          data_source: forward_tacotron_melgan_io
+          reader: numpy_dict_reader
+          annotation_conversion:
+            converter: multi_feature_regression
+            data_dir: forward_tacotron_melgan_io
+            input_suffix: duration_prediction_in_
+            reference_suffix: melgan_out_
```

Comment:
could you please move dataset specific things to data/dataset_definitions.yml and keep only name in config?

Reply:
done